### PR TITLE
NCTL : Gas payment fix + config update

### PR DESCRIPTION
**Pull request by ghost**
1. Updated default gas payments to pass latest crypto-economic requirements.
2. Removed use of nctl specific templates.
3. New commands to view node config + chainspec.
4. Used less rather than vi when opening files for inspection.

### CODE REVIEW

File: utils/nctl/sh/utils/misc.sh [hunk 1]
```diff
@@ -120,170 +177,137 @@ function resetd () {
...
 }
 
 #######################################
-# Returns node RPC address, intended for use with cURL.
+# Get a network known addresses - i.e. those of bootstrap nodes.
 # Arguments:
-#   Network ordinal identifier.
-#   Node ordinal identifier.
+#   Network ordinal identifer.
+#   Node ordinal identifer.
```
Comment (Fraser999) [inline utils/nctl/sh/utils/misc.sh:213]: I prefer the old spellings :)

**Review by Fraser999 — APPROVED** [review]
I haven't gone through the PR with a fine-toothed comb, but it generally looks good to me.

The change away from using templates for the config and chainspec is particularly welcome.

File: utils/nctl/sh/assets/setup.sh [hunk 2]
```diff
@@ -25,19 +25,17 @@ function _set_bin() {
...
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/pos_install.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/standard_payment_install.wasm $1/bin
-    # ... client contracts.
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/add_bid.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/delegate.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/transfer_to_account_u512.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/transfer_to_account_u512_stored.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/undelegate.wasm $1/bin
-    cp $NCTL_CASPER_HOME/target/wasm32-unknown-unknown/release/withdraw_bid.wasm $1/bin
+    # Set system contracts.
```
Comment (linclelinkpart5) [inline utils/nctl/sh/assets/setup.sh:28]: I like this change!

File: utils/nctl/sh/assets/setup.sh [hunk 3]
```diff
@@ -53,17 +51,17 @@ function _set_chainspec() {
...
     path_config=$1/chainspec/chainspec.toml
-    cp $NCTL/templates/chainspec.toml $path_config
-    sed -i "s/{GENESIS_NAME}/$GENESIS_NAME/g" $path_config > /dev/null 2>&1
-    sed -i "s/{GENESIS_TIMESTAMP}/$GENESIS_TIMESTAMP/g" $path_config > /dev/null 2>&1
-    sed -i "s/{HIGHWAY_GENESIS_ERA_START_TIMESTAMP}/$HIGHWAY_GENESIS_ERA_START_TIMESTAMP/g" $path_config > /dev/null 2>&1
+    cp $NCTL_CASPER_HOME/resources/local/chainspec.toml $path_config
+
+    # Set config setting: genesis.name.
+    GENESIS_NAME=casper-net-$2
+    sed -i "s/casper-example/$GENESIS_NAME/g" $path_config > /dev/null 2>&1
```
Comment (linclelinkpart5) [inline utils/nctl/sh/assets/setup.sh:60]: Could matching on "casper-example" break in the future? If it were to get renamed, we'd no longer do the substitution properly

**Review by linclelinkpart5 — APPROVED** [review]

File: utils/nctl/sh/assets/setup.sh (hunk 3 above)
Comment (momipsl) [reply utils/nctl/sh/assets/setup.sh:60]: Yeah it is a little brittle.  Might try to improve it - on another day.

**Pull request merged**